core/templates/dev/head/pages/exploration_player/AudioControlsDirective.js

```diff
@@ -32,15 +32,15 @@ oppia.directive('audioControls', [
       controller: [
         '$scope', 'AudioTranslationManagerService', 'AudioPlayerService',
         'LanguageUtilService', 'AssetsBackendApiService',
-        'AutogeneratedAudioPlayerService',
+        'AutogeneratedAudioPlayerService', 'PlayerPositionService',
         function(
             $scope, AudioTranslationManagerService, AudioPlayerService,
             LanguageUtilService, AssetsBackendApiService,
-            AutogeneratedAudioPlayerService) {
+            AutogeneratedAudioPlayerService, PlayerPositionService) {
           // This ID is passed in to AudioPlayerService as a means of
           // distinguishing which audio directive is currently playing audio.
           var directiveId = Math.random().toString(36).substr(2, 10);
-
+          var requestedToPlayAudio = false;
```

Comment:
Maybe audioRequestIsPending, since you set this to false once the audio starts playing?

Reply:
Done